features/openapi/docs.feature

```diff
@@ -95,7 +95,7 @@ Feature: Documentation support
     And the JSON node "paths./related_dummies/{id}/related_to_dummy_friends.get.parameters" should have 6 elements
 
     # Subcollection - check schema
-    And the JSON node "paths./related_dummies/{id}/related_to_dummy_friends.get.responses.200.content.application/ld+json.schema.properties.hydra:member.items.$ref" should be equal to "#/components/schemas/RelatedToDummyFriend:jsonld-fakemanytomany"
+    And the JSON node "paths./related_dummies/{id}/related_to_dummy_friends.get.responses.200.content.application/ld+json.schema.properties.hydra:member.items.$ref" should be equal to "#/components/schemas/RelatedToDummyFriend%3Ajsonld-fakemanytomany"
```

Comment:
If we introduce a BC break, shouldn't we also change to a URL-safe character as separator to make the names more human-readable?

Reply:
Currently there should be no BC break as only the refs are affected, not the definition names itself.
And you prefered urlencode in https://github.com/api-platform/core/issues/1969#issuecomment-391735278
But sure we can switch to safe chars, if you like. Which do you prefer? I guess `.` would be a good alternative to `:`?